app/jsx/pages/BrowsePage.jsx

```diff
@@ -77,11 +165,15 @@ class BrowseJournals extends React.Component {
 //       :ancestor_unit=>["ucla", "ucsc"],
 //       :is_active=>true}
   getVisibleJournals(journals, isActive, campusID) {
-    return journals.map(function(j, i) {
-      return (j['is_active'] == isActive || isActive == "") &&
+    let foundOne = false
+    let r = journals.map(function(j, i) {
```

Comment:
Just wondering whether there's any value to the server returning inactive journals at all. If the server filtered them out, they'd be unnecessary here.

Or maybe the thinking was, if the user is logged as an admin, then you'd show the inactive journals? That could make some sense.